### DISCUSSION TITLE
Release of version v0.9.0

### DISCUSSION THREAD

**Discussion by DanielChappuis** [DanielChappuis/reactphysics3d#233] [Announcements]
I am proud to announce that the version [v0.9.0](https://github.com/DanielChappuis/reactphysics3d/releases/tag/v0.9.0) of the library has just been released.

Do not hesitate to take a look at the release notes to see what are the new features and what has been changed or fixed.

I hope you will enjoy this version.

**Comment by kencb** [comment]
Awesome! Really glad to see that all the warnings should be fixed, and everything else too ofc.

**Comment by kveale** [comment]
Really liking the library. Thanks for your work on this. Would like to see the ability to query the world with an AABB in addition to the ray query.
